clang/tools/clang-repl/ClangRepl.cpp

```diff
@@ -16,13 +16,24 @@
 #include "clang/Interpreter/CodeCompletion.h"
 #include "clang/Interpreter/Interpreter.h"
 
+#include "llvm/ADT/StringExtras.h"
+#include "llvm/ExecutionEngine/Orc/ExecutorProcessControl.h"
 #include "llvm/ExecutionEngine/Orc/LLJIT.h"
+#include "llvm/ExecutionEngine/Orc/SimpleRemoteEPC.h"
 #include "llvm/LineEditor/LineEditor.h"
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/ManagedStatic.h" // llvm_shutdown
 #include "llvm/Support/Signals.h"
 #include "llvm/Support/TargetSelect.h"
+#include "llvm/TargetParser/Host.h"
 #include <optional>
+#include <sys/types.h>
+
+#ifdef LLVM_ON_UNIX
+#include <netdb.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#endif // LLVM_ON_UNIX
```

Comment:
Have you had a chance to try it out on Windows? If that's beyond our scope we might want to add the platform to the unsupported platforms on the tests.

Reply:
I was seeing build failures in CI due to some headers. `llvm-jitlink` (Where I got most of the code for creating/managing the executor process) does not support connecting to an already running executor over TCP.